src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -287,6 +299,7 @@ public String getVersion() {
      * @param stream Where to write the usage message.
      */
     public void usage(final PrintStream stream, final boolean printCommon) {
+    	
```

Comment:
The other functions in this file don't have a line after the declaration, no need to add one here